### PR TITLE
feat: Added Avatar and Badges

### PR DESCRIPTION
**Pull request by arjunbharti**
Created the following components:
    
    1. Avatar
    
    2. Badges
        2.1. Badge on Avatars
        2.2. Badge on Icons

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.display-badge{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
```
Comment (xii-vi) [inline components/badge.css:4]: You use the same property in the avatar, you could make this utility class and you wherever required

**Review by xii-vi — APPROVED** [review]
A tiny bit of suggestion in the comment. You can merge after that.

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    background-color: red;
+}
+
+.badge-green{
+    background-color: green;
+}
+
+.badge-pink{
+    background-color: pink;
+}
```
Comment (shrushti2000) [inline components/badge.css]: try using CSS variables.

Comment (arjunbharti) [reply components/badge.css]: Updated successfully!

File: components/badge.css (hunk 1 above)
Comment (arjunbharti) [reply components/badge.css:4]: Currently, I am not following the utility first approach but I will use it once I start writing grid components. Thanks for the suggestion!

**Pull request merged**